style.css

```diff
@@ -0,0 +1,121 @@
+* {
```

Comment:
- [ ] You haven't integrated fonts correctly with your project. Kindly follow this [video](https://www.loom.com/share/36f4fbc966ce43b0a9d76daca5e4b430) to add font correctly.

Reply:
@Tufahel 
please re review my project for the changes because I only have 1 review left